18/README.md

```diff
@@ -42,16 +42,28 @@ A Validator must be able to discern valid Votes from invalid ones. A valid vote
 ### What is a valid Election?
 A Validator must be able to discern valid Elections from invalid ones. A valid Election is a transaction where **all** the following conditions are true.
 
-1. `operation` is `CREATE`, or equivalent.
-1. The `inputs` satisfy the conditions for a CREATE transaction. For example, the public key listed in `inputs.owners_before` is the public key of the Initiator.
+1. The transaction complies to the `CREATE` transaction JSON schema.
+1. The transaction additionally complies to the [election JSON schema](#./election_transaction.yaml).
+1. `operation` has a value specific to the particular election, e.g. `VALIDATOR_ELECTION`.
 1. `outputs` has as many entries as the total number of Validators.
 1. Each Validator is represented in `outputs`.
 1. Each entry in `outputs` can be spent by only one Validator, and the amount attached to it is equal to the power of that Validator.
 
-**Note: any change in the Validator Set will make old Elections invalid. Check [approach 2](#generalized-approach-approach-2) for a process that can tolerate a certain degree of change to the Validator Set.**
+**Note: any change in the Validator Set makes old Elections invalid. Check [approach 2](#generalized-approach-approach-2) for a process that can tolerate a certain degree of change to the Validator Set.**
+
+### Election statuses
+The lifecycle of an Election is described by the three statuses:
+
+- `ongoing`
+- `concluded`
+- `inconclusive`
+
+`ongoing` are valid Elections denoted by committed transactions, which did not receive a sufficient amount of votes, and the validator set has not changed since their creation.
+
+Elections are considered `inconclusive` if they had not been concluded by the time the validator set was changed.
```

Comment:
For the sake of completeness, one line for `concluded` status could also be added to the spec.